docs/integrations/google/workspace/collect-logs.md

```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
 **Google Workspace Alert Center** alerts are in JSON format. Most of the alerts have a few common fields. The differences are in the data section of the JSON where the alert type specific details are recorded. For more information, see this Google Workspace [Alert document](https://developers.google.com/admin-sdk/alertcenter/reference/alert-types).
 
 
-## Configure Collection for Google Workspace Audit Source
+## Configure Collection for Google Workspace Source
 
-This section provides instructions for configuring log collection for Google Workspace with Audit Source.
+This section provides instructions for configuring log collection for Google Workspace Source.
 
-[Google Workspace](#configure-collection-for-google-workspace-audit-source): Monitors and analyzes the activity across all the Google Workspace Apps in one place. You can configure collection for each Google App for which you want to analyze events:
+[Google Workspace](#configure-collection-for-google-workspace-source): Monitors and analyzes the activity across all the Google Workspace Apps in one place. You can configure collection for each Google App for which you want to analyze events:
     * Google Admin
     * Google Drive
     * Google Login
     * Google Token
 
 ### About Source Configuration  
 
-Currently, the source name for Google Workspace is still **G Suite Apps Audit Source**, which will be changed/updated shortly.
-
-Configure one [G Suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source) for each Google App from which you want to collect events:
```

Comment:
I don't know when we will rename this source ( to "Google Workspace Audit Source"?), right now it's shown as "G Suite Apps Audit Source"